### PR TITLE
Fix audit docs

### PR DESCRIPTION
**Pull request by johannesfrey**
https://deploy-preview-162--vcluster-docs-site.netlify.app/docs/platform/manage/audit

Resolves DOC-116

### CODE REVIEW

File: platform/manage/audit.mdx [hunk 1]
```diff
@@ -178,9 +178,9 @@ You cannot configure this under Admin > Config, since this requires a change in
 Then update the helm release via:
 
 ```
-helm upgrade loft vcluster-control-plane --namespace vcluster-platform \
+helm upgrade vcluster-platform vcluster-platform --namespace vcluster-platform \
                                          --repo https://charts.loft.sh \
-                                         --version $LOFT_VERSION \
+                                         --version 4.0.0-alpha.12 \
```
Comment (heiko-braun) [inline platform/manage/audit.mdx:183]: I think we should switch this to a placeholder like `<PLATFORM_VERSION>`

**Review by heiko-braun — CHANGES_REQUESTED** [review]
Maybe

File: platform/manage/audit.mdx (hunk 1 above)
Comment (deniseschannon) [reply platform/manage/audit.mdx:183]: Agreed on this approach.

Comment (heiko-braun) [reply platform/manage/audit.mdx:183]: Well, the downside is that it's not executable anymore if people copy/paste

Comment (heiko-braun) [reply platform/manage/audit.mdx:183]: Let's get the naming changes in and discuss the variable substitution here https://github.com/loft-sh/vcluster-docs/pull/164 – it suffers from the same problem

**Review by heiko-braun — APPROVED** [review]

**Pull request merged**